### PR TITLE
Fix unpacking update request 

### PR DESCRIPTION
**Pull request by LiliC**
This should fix https://github.com/pmorie/osb-broker-lib/issues/19

### CODE REVIEW

**Comment by n3wscott** [comment]
LGTM

**Comment by shawn-hurley** [comment]
Should we add test cases to verify that the correct data is getting passed through?

File: pkg/rest/apisurface.go [hunk 1]
```diff
@@ -420,15 +420,17 @@ func (s *APISurface) UpdateHandler(w http.ResponseWriter, r *http.Request) {
...
+	}
 
 	vars := mux.Vars(r)
-	osbRequest.ServiceID = vars[osb.VarKeyServiceID]
+	osbRequest.InstanceID = vars[osb.VarKeyInstanceID]
 
-	planID := vars[osb.VarKeyPlanID]
-	if planID != "" {
-		osbRequest.PlanID = &planID
+	asyncQueryParamVal := r.FormValue(osb.AcceptsIncomplete)
```
Comment (jmrodri) [inline pkg/rest/apisurface.go]: :+1:

**Comment by LiliC** [comment]
@shawn-hurley Agreed! Adding them now :)

**Comment by pmorie** [comment]
@LiliC tells me she's time constrained and won't be able to finish the tests for a couple days. Merging now since I trust her to follow through :)

**Pull request merged**